.github/workflows/ci.yml

```
@@ -65,6 +65,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-dotnet@v1
+      - run: dotnet build
```

Shouldn't this one have a name too?
```suggestion
      - name: Prebuild for protobuc plugin
      - run: dotnet build
```